test_collections/matter/scripts/OTBR/otbr_start.sh

```diff
@@ -1,24 +1,25 @@
 #! /usr/bin/env bash
-
- #
- # Copyright (c) 2023 Project CHIP Authors
- #
- # Licensed under the Apache License, Version 2.0 (the "License");
- # you may not use this file except in compliance with the License.
- # You may obtain a copy of the License at
- #
- # http://www.apache.org/licenses/LICENSE-2.0
- #
- # Unless required by applicable law or agreed to in writing, software
- # distributed under the License is distributed on an "AS IS" BASIS,
- # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- # See the License for the specific language governing permissions and
- # limitations under the License.
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 ROOT_DIR=$(realpath $(dirname "$0")/../../../../..)
 TH_SCRIPTS_DIR="$ROOT_DIR/scripts"
 
 DEFAULT_OTBR_INTERFACE="eth0"
 BR_INTERFACE=${1:-$DEFAULT_OTBR_INTERFACE}
+BR_VARIANT="35" TODO: Receive the BR variant via shell script parameter
```

Comment:
```suggestion
BR_VARIANT="35" # TODO: Receive the BR variant via shell script parameter
```

Reply:
Thanks for pointing it out. Almost missed.

Done in https://github.com/project-chip/certification-tool-backend/pull/150/commits/89fe412e9363a20cf8e2658b0f5f633115924d77